File: java/webflux-security-example/src/main/java/com/van1164/webfluxsecurityexample/WebfluxSecurityExampleApplication.java
package com.van1164.webfluxsecurityexample;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.PropertySource;

@SpringBootApplication
@PropertySource("classpath:application.properties")
public class WebfluxSecurityExampleApplication {

	public static void main(String[] args) {

		SpringApplication.run(WebfluxSecurityExampleApplication.class, args);
	}

}

File: kotlin/webflux-security-example/src/main/kotlin/com/van1164/webfluxsecurityexample/WebfluxSecurityExampleApplication.kt
package com.van1164.webfluxsecurityexample

import mu.KotlinLogging
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class WebfluxSecurityExampleApplication

val logger = KotlinLogging.logger {  }

fun main(args: Array<String>) {
	runApplication<WebfluxSecurityExampleApplication>(*args)
}
